.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=true
```

Comment:
The value of `package-lock` is true by default.
https://docs.npmjs.com/cli/v10/using-npm/config#package-lock

Reply:
> The value of `package-lock` is true by default. https://docs.npmjs.com/cli/v10/using-npm/config#package-lock

If the user overrides the default value of package-lock in the global configuration, the lock file will not be generated properly. 

Therefore, my recommendation is to set the default value of package-lock in the project